docs/rc.xml.all

```diff
@@ -380,7 +380,7 @@
     <windowRule identifier="*">
       <action name="Maximize" />
     </windowRule>
-    <windowRule identifier="some-application" serverDecoration="yes" />
+    <windowRule title="some-title" serverDecoration="yes" />
   </windowRules>
```

Comment:
Should we add examples for all combinations? E.g
- only identifier (set to `*`, so the first example rule)
- identifier and title set
- only title set

Reply:
Have added one of each + both.